File: app/src/main/java/com/artmurka/artmurkaapp/Model/InterfacesModel/IWishList.java
package com.artmurka.artmurkaapp.Model.InterfacesModel;

import com.artmurka.artmurkaapp.Model.Pojo.ItemList.WishList.WishList;

import retrofit2.Call;

public interface IWishList {
    Call<WishList> toWishList(String goods_id);
}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/interfacesmodel/IWishList.kt
package com.artmurka.artmurkaapp.data.model.interfacesmodel

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.wishList.WishList
import io.reactivex.Observable

import retrofit2.Call

interface IWishList {
    val wishList: Observable<WishList>
    fun toWishList(goods_id: String): Observable<WishList>
}
